src/components/inputs/Search/SearchBar/SearchBar.tsx

```diff
@@ -123,86 +132,114 @@ function SearchBar(props: SearchBarProps) {
+                  ALPHA_NUMERIC_REGEX.test(query[query.length - 1])
+                    ? resolve(true)
+                    : reject(),
+                ),
+            },
+          }}
+          customValidationIcons={{
+            error: (
+              <CheckCircleFilled
+                style={{ fontSize: 16, color: 'var(--text-faded)' }}
+              />
+            ),
+            success: (
+              <CheckCircleFilled
+                style={{ fontSize: 16, color: 'var(--success-green)' }}
+              />
+            ),
           }}
-          ref={inputRef}
-          value={searchBarText ?? ''}
-          maxLength={20}
-          className="searchbar-input"
-          style={height ? { height: `${height}px` } : {}}
         />
         {isMobile ? (
           <></>
         ) : (
-          <>
-            {!isAvailable || !searchBarText || !searchSubmitted ? (
-              <button
-                className="search-button"
-                style={{
-                  width: `${height}px`,
-                  height: `${height}px`,
-                  maxWidth: `${height}px`,
-                  maxHeight: `${height}px`,
-                }}
-                onClick={() => {
-                  isSearchValid && _onSubmit();
-                }}
-              >
-                <SearchIcon
-                  fill="#121212"
-                  stroke="white"
-                  width={height ? `${height / 2.2}` : '18.51px'}
-                  height={height ? `${height / 2.2}` : '18.51px'}
-                />
-              </button>
-            ) : (
-              <>
-                <span
-                  className="test faded bold"
-                  style={{ marginRight: '18px' }}
-                >
-                  Register
-                </span>
-                <button
-                  className="accent icon-button"
-                  onClick={_onSubmitButton}
-                >
-                  <ArrowUpRight
-                    fill="var(--text-black)"
-                    stroke="var(--text-black)"
-                    width="18.51"
-                    height="18.51"
-                  />
-                </button>
-              </>
-            )}
-          </>
+          <button
+            className="button pointer"
+            style={{
+              width: `${height}px`,
+              height: `${height}px`,
+              borderLeft: '1px solid #38393B',
+            }}
+            onClick={() => {
+              isSearchValid && _onSubmit();
+            }}
+          >
+            <SearchIcon fill="white" width={18} height={18} />
+          </button>
         )}
       </div>
+      {searchSubmitted && isAvailable ? (
+        <button
+          className="accent-button center"
+          onClick={_onSubmitButton}
+          style={{
+            width: 130,
+            height: 50,
+            fontSize: 14,
+            padding: 0,
+            marginTop: 30,
```

Comment:
why can't we use an existing button class for this? also - noticeably moves down the feature domain on the page when this shows. not a stopper - but seems glitchy

Reply:
the moving to featured domains is actually part of the figma - that state of the searchbar pushed it down while maintaining the gap.

Updated the class to be more generic with common styles.

changes in https://github.com/ar-io/pdns-react/pull/227/commits/86a8620bc0107969766621a5438b2912e2f1cf82